### ISSUE TITLE
longrunning: cannot build longrunning since v0.3.0

### ISSUE THREAD

**Issue by darist** [googleapis/google-cloud-go#7233]
**Client**

longrunning

**Environment**

linux (using bazel)

**Go Environment**

$ go version

go version go1.19.4 linux/amd64

$ go env

GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/darist/.cache/go-build"
GOENV="/home/darist/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/darist/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/darist/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go-1.19"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go-1.19/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.19.4"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/darist/tmp/datainfra-beagle-fusion/go.mod"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build3973921892=/tmp/go-build -gno-record-gcc-switches"

**Code**

```shell
$ bazel build @com_google_cloud_go_longrunning//:longrunning
```

**Expected behavior**

Expect build to succeed.

**Actual behavior**

```shell
$ bazel build @com_google_cloud_go_longrunning//:longrunning
INFO: Analyzed target @com_google_cloud_go_longrunning//:longrunning (3 packages loaded, 9 targets configured).
INFO: Found 1 target...
ERROR: /home/darist/.cache/bazel/_bazel_darist/5e6c2edf559fe4050cd372ccd505374b/external/com_google_cloud_go_longrunning/BUILD.bazel:3:11: GoCompilePkg external/com_google_cloud_go_longrunning/longrunning.a failed: (Exit 1): builder failed: error executing command (from target @com_google_cloud_go_longrunning//:longrunning) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder compilepkg -sdk external/go_sdk -installsuffix linux_amd64 -src external/com_google_cloud_go_longrunning/longrunning.go -embedroot '' ... (remaining 32 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
external/com_google_cloud_go_longrunning/longrunning.go:60:10: cannot use inner (variable of type *"cloud.google.com/go/longrunning/autogen".OperationsClient) as type operationsClient in struct literal:
        *"cloud.google.com/go/longrunning/autogen".OperationsClient does not implement operationsClient (wrong type for CancelOperation method)
                have CancelOperation(ctx context.Context, req *longrunningpb.CancelOperationRequest, opts ...gax.CallOption) error
                want CancelOperation(context.Context, *"google.golang.org/genproto/googleapis/longrunning".CancelOperationRequest, ...gax.CallOption) error
compilepkg: error running subcommand external/go_sdk/pkg/tool/linux_amd64/compile: exit status 2
Target @com_google_cloud_go_longrunning//:longrunning failed to build
Use --verbose_failures to see the command lines of failed build steps.
```

**Screenshots**

![image](https://user-images.githubusercontent.com/16511946/211636662-b31dffff-1616-4949-bc36-6b86d2e9a7f9.png)

**Additional context**

I think the problem is that `longrunning.go` imports `"google.golang.org/genproto/googleapis/longrunning"`,
but `longrunning/autogen/operations_client.go` (which used to import the same) now imports `"cloud.google.com/go/longrunning/autogen/longrunningpb"`,

Looks to me like it was introduced by #6951 and first released in longrunning@v0.3.0

**Comment by codyoss** [comment]
Hey,

We recently underwent a migration where we moved the proto stubs from genproto into this repo. Although the import path in longrunning.go should indeed be updated if we look at a newer version of genproto we will we that the protos there are just aliases back to this repo: https://github.com/googleapis/go-genproto/blob/76db0878b65f00c3f93403b4b9a155af313cc4ba/googleapis/longrunning/alias.go#L26

In the Go module world this is all resolved for you, but it looks like bazel builds likely need to pin newer versions themselves. 

I will make a PR to update the import though as they both are the same thing. Hope that helps.

**Comment by darist** [comment]
Thanks, @codyoss! I ended up pinning my dep to v0.2.1 as a work-around for now.  I tried re-arranging my bazel WORKSPACE file to try to get the latest version of genproto, and that got longrunning building for me... but then something else (unrelated) broke. I just couldn't get the order of operations in the WORKSPACE file just right to get everything working, so falling back on v0.2.1 was an easy patch for now.
